examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.php

```diff
@@ -0,0 +1,454 @@
+
+        return $addedCampaignResourceName;
+    }
+
+    /**
+     * Creates an ad group for the remarketing campaign.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param string $campaignResourceName the resource name of the campaign that
+     *     the new ad group will belong to
+     * @return string the resource name of the newly created ad group
+     */
+    private static function createAdGroup(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $campaignResourceName
+    ) {
+        // Creates the ad group.
+        $adGroup = new AdGroup([
+            'name' => new StringValue(['value' => 'Dynamic remarketing ad group']),
+            'campaign' => new StringValue(['value' => $campaignResourceName]),
+            'status' => AdGroupStatus::ENABLED
+        ]);
+
+        // Creates an ad group operation.
+        $adGroupOperation = new AdGroupOperation();
+        $adGroupOperation->setCreate($adGroup);
+
+        // Issues a mutate request to add the ad group.
+        $adGroupServiceClient = $googleAdsClient->getAdGroupServiceClient();
+        $response = $adGroupServiceClient->mutateAdGroups($customerId, [$adGroupOperation]);
+
+        /** @var AdGroup $addedAdGroup */
+        $addedAdGroup = $response->getResults()[0];
+        $addedAdGroupResourceName = $addedAdGroup->getResourceName();
+        printf("Created ad group with resource name '%s'.%s", $addedAdGroupResourceName, PHP_EOL);
+
+        return $addedAdGroupResourceName;
+    }
+
+    /**
+     * Creates the responsive display ad.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param string $adGroupResourceName the resource name of the ad group that
+     *     the new ad group ad will belong to
+     */
+    private static function createAd(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $adGroupResourceName
+    ) {
+        $marketingImageResourceName = self::uploadAsset(
+            $googleAdsClient,
+            $customerId,
+            'https://goo.gl/3b9Wfh',
+            'Marketing Image'
+        );
+        $logoImageResourceName = self::uploadAsset(
+            $googleAdsClient,
+            $customerId,
+            'https://goo.gl/mtt54n',
+            'Logo Image'
+        );
+
+        // Creates the responsive display ad info object.
+        $responsiveDisplayAdInfo = new ResponsiveDisplayAdInfo([
+            'marketing_images' => [
+                new AdImageAsset([
+                    'asset' => new StringValue(['value' => $marketingImageResourceName])
+                ])
+            ],
+            'square_marketing_images' => [
+                new AdImageAsset([
+                    'asset' => new StringValue(['value' => $logoImageResourceName])
+                ])
```

Comment:
A bit weird that logo is used as square marketing image...
Can you put a comment explaining here (i.e., for demonstrating purposes; it doesn't need to be the same as logo)?